### ISSUE TITLE
No Fantasy points for 2nd day

### ISSUE THREAD

**Issue by nyxulric1902** [ValveSoftware/Dota2-Gameplay#12799]
### Description

Havent received any points to my fantasy roster for the 2nd day.

**Comment by driverog** [comment]
Same here

**Comment by commentator24** [comment]
you dont get points for every match, only best score used ![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/109324155/ad5c7ab1-42d5-4313-9b18-4e68422ad6d5)

**Comment by SoftPancakes42** [comment]
> you dont get points for every match, only best score used ![image](https://user-images.githubusercontent.com/109324155/275138545-ad5c7ab1-42d5-4313-9b18-4e68422ad6d5.png)

Yeah but it seems a bit insane that the 'best score' from people in the top 100 is insanely high in comparison to everyone else. The number of points just seems unreasonable in comparison to the first day. At the end of day 1 people were sitting on 55-80k points, now they have nearly double that.

**Comment by nyxulric1902** [comment]
> you dont get points for every match, only best score used ![image](https://user-images.githubusercontent.com/109324155/275138545-ad5c7ab1-42d5-4313-9b18-4e68422ad6d5.png)

you get points every day for the best two games in the day.. i didnt get points on day 2

**Comment by LgnAA** [comment]
Same) I miscalculated the amount on the first day. Now they seem to have added up my points correctly for the first day, but they didn’t give me the points for the second

**Comment by SoftPancakes42** [comment]
> > you dont get points for every match, only best score used ![image](https://user-images.githubusercontent.com/109324155/275138545-ad5c7ab1-42d5-4313-9b18-4e68422ad6d5.png)
> 
> you get points every day for the best two games in the day.. i didnt get points on day 2

That depends on the definition of a series...and period. If they say that a period is the whole weekend - then we get points for a total of TWO games. That's it because it will be the 'best scoring series' used. Additionally, this means that in comparison to day 1 the fantasy rosters of people who were in first place have performed nearly twice as well on day two in comparison to their day 1 results. Doesn't that seem a bit of a jump to make? 

To have essentially no alterations to a score means that a team has played the best they have so far and those players have not outperformed their metrics. To say that a fantasy roster is able to double its points because players are doing astronomically better and only take two games into consideration seems mathematically improbable.

**Comment by nyxulric1902** [comment]
> > > you dont get points for every match, only best score used ![image](https://user-images.githubusercontent.com/109324155/275138545-ad5c7ab1-42d5-4313-9b18-4e68422ad6d5.png)
> > 
> > 
> > you get points every day for the best two games in the day.. i didnt get points on day 2
> 
> That depends on the definition of a series...and period. If they say that a period is the whole weekend - then we get points for a total of TWO games. That's it because it will be the 'best scoring series' used. Additionally, this means that in comparison to day 1 the fantasy rosters of people who were in first place have performed nearly twice as well on day two in comparison to their day 1 results. Doesn't that seem a bit of a jump to make?
> 
> To have essentially no alterations to a score means that a team has played the best they have so far and those players have not outperformed their metrics. To say that a fantasy roster is able to double its points because players are doing astronomically better and only take two games into consideration seems mathematically improbable.

i am saying the same thing as you lmao... that we didnt get points for day 2

**Comment by SoftPancakes42** [comment]
> > > > you dont get points for every match, only best score used ![image](https://user-images.githubusercontent.com/109324155/275138545-ad5c7ab1-42d5-4313-9b18-4e68422ad6d5.png)
> > > 
> > > 
> > > you get points every day for the best two games in the day.. i didnt get points on day 2
> > 
> > 
> > That depends on the definition of a series...and period. If they say that a period is the whole weekend - then we get points for a total of TWO games. That's it because it will be the 'best scoring series' used. Additionally, this means that in comparison to day 1 the fantasy rosters of people who were in first place have performed nearly twice as well on day two in comparison to their day 1 results. Doesn't that seem a bit of a jump to make?
> > To have essentially no alterations to a score means that a team has played the best they have so far and those players have not outperformed their metrics. To say that a fantasy roster is able to double its points because players are doing astronomically better and only take two games into consideration seems mathematically improbable.
> 
> i am saying the same thing as you lmao... that we didnt get points for day 2

Yeah, it's really sucky :( 
I dislike it.

**Comment by Kk-Ling** [comment]
same

**Comment by Jadcc** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/12794#issuecomment-1762821889

**Comment by Jadcc** [comment]
> you dont get points for every match, only best score used ![image](https://user-images.githubusercontent.com/109324155/275138545-ad5c7ab1-42d5-4313-9b18-4e68422ad6d5.png)

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/147919666/a02b2f82-7559-4c7a-8b07-26207e780c88)
40k on day 1 and still the same points on day 2
ps. on day 2 i saw yatoro score having 20k from other people but mine only have 14k so his best score is 20k not 14k this is a bug i think pls fix this

id:156579776

**Comment by jonasmarte** [comment]
same here, score did not move when I crafted playoff war banners

<img width="992" alt="image" src="https://github.com/ValveSoftware/Dota2-Gameplay/assets/59923143/22c3e5ef-0d40-4f6a-aa31-4133e9dc7240">
